app/routes/project/settings/contributors.js

```diff
@@ -4,7 +4,6 @@ const { Route } = Ember;
 
 export default Route.extend({
   model() {
-    let project = this.modelFor('project');
-    return project;
+    return this.modelFor('project');
```

Comment:
`project.settings`is already reloading, so I don't think `project.settings.contributors` has to, but I'm not sure.